src/tlo/methods/alri.py

```diff
@@ -2595,6 +2597,8 @@ def _get_disease_classification_for_treatment_decision(self,
                  'chest_indrawing_pneumonia',       (symptoms-based assessment)
                  'cough_or_cold'                    (symptoms-based assessment)
          }."""
+        # TODO: Currently this is logged as equipment even if pulse ox consumable isnt available
+        self.EQUIPMENT.update({'Pulse oximeter'})
```

Comment:
Why is the equipment updated even if consumable needed isn't available? Where is the consumable availability verified?

And from the code below, do I understand correctly that the oximeter should be only used if `use_oximeter` is `True`?

Reply:
if it is exact same thing it should be only as consumable not doubled as equipment

Reply:
leaving this as it is whilst we sort out consumable/equipment overlap issue